### ISSUE TITLE
Only Ctrl + Alt + Del works on the Apple II

### ISSUE THREAD

**Issue by hackwin** [mafrmt00/Apple_II_Keyboard#3]
I followed the instructions and only get the reset command to work on the Apple II.

None of the keys are showing on the Apple II screen.

![javaw_Xbx8DQdG3n](https://user-images.githubusercontent.com/1328566/136708812-29725dea-3eb7-4783-8827-7040a2773555.png)
![IMG_6123](https://user-images.githubusercontent.com/1328566/136708829-8617b116-f101-4016-b73e-76a559675de4.JPG)

**Comment by mafrmt00** [comment]
The Arduino Terminal looks good.
Could you measure the voltage on the data pins and double check the wiring.

**Comment by hackwin** [comment]
Thanks.  I hooked up the Apple II directly to the Arduino.  It worked without the USB Host Shield.  Not really sure what the deal is.

```
#include "II_Encoder.h"
II_Encoder KeyEncoder;

void setup() {
  Serial.begin( 115200 );
  KeyEncoder.begin();
}

void loop() {
  char incomingByte = 0;

   while(Serial.available() > 0){
    incomingByte = Serial.read();
    if(incomingByte != '\n'){
      KeyEncoder.IIputchar(incomingByte);
      delay(10);
    }

    if(incomingByte == 0x1E){
      KeyEncoder.Reset();
      delay(10);
    }
    
   }
   
}
```

Maybe you want to add the above code as an example.  It works with Arduino Serial, RealTerm, and PuTTY.

**Comment by mafrmt00** [comment]
The USB Host Shield has some jumpers are they set correct? The shield I used did not interfere with the data pins used in the sketch, is yours the same?

**Comment by hackwin** [comment]
I believe the jumpers are the same.  Please take a look at the photo below.

![IMG_6142](https://user-images.githubusercontent.com/1328566/137606025-5258c91e-1c4a-4c36-9667-3e6414ad0b07.JPG)

I ordered another, different USB host shield adapter and will test that when it arrives.

**Comment by hackwin** [comment]
So I ran another test with the following code: 

```
void setup() {
  Serial.begin(115200);
  for(int i=3; i<9; i++){
    pinMode(i, INPUT);
  }
  for(int i=A0; i<A4; i++){
    pinMode(i, INPUT);
  }
  pinMode(13, OUTPUT);
  digitalWrite(13, HIGH);
}

void loop() {
  for(int i=3; i<9; i++){
    Serial.println(String("pin ") + String(i) + String(": ") + String(digitalRead(i)));
  }
  for(int i=A0,j=0; i<A4,j<4; i++,j++){
    Serial.println(String("pin A") + String(j) + String(": ") + String(digitalRead(i)));
  }
  Serial.println("-----------------------");
  delay(1000);
}
```

Pin 4 didn't read "1" (HIGH) through the USB Host Shield when connected to 5v using a jumper.  

I edited the pins in II_Encoder.h as follows:

```
#define IIKEYB_RESET 3
#define IIKEYB_STRB 2
```

Now it's working.  I plan to merge the code from Serial input so a file can be emulated as though it's being typed.

![realterm_VggrTea02Q](https://user-images.githubusercontent.com/1328566/137631374-f9015b90-9066-4d64-802b-14880fa9dc05.png)
![notepad_gnM3SqnWLO](https://user-images.githubusercontent.com/1328566/137631398-31a77bb7-fc5c-4a89-8cec-04115305290b.png)
![IMG_6146](https://user-images.githubusercontent.com/1328566/137631405-a8d3cefd-093f-4bbb-95fb-dea46f407563.JPG)

Basically it's emulating typing the file into the keyboard port really fast.

**Comment by mafrmt00** [comment]
![Arduino-USB-2 0-Host-shield-schema](https://user-images.githubusercontent.com/11454230/137677558-372ec6c7-e0ed-48d8-8e7c-798c65ebf36f.png)
Congratulations!
Pin 4 should not be influenced by the USB-HOST Shield in any way!
